example/gardener-local/gardenlet/values-provider-extensions.yaml

```diff
@@ -0,0 +1,73 @@
+config:
+  gardenClientConnection:
+    bootstrapKubeconfig:
+      kubeconfig: |
+        apiVersion: v1
+        kind: Config
+        current-context: gardenlet-bootstrap
+        clusters:
+        - cluster:
+            insecure-skip-tls-verify: true
+            server: https://gardener-apiserver.relay.svc.cluster.local
+          name: default
+        contexts:
+        - context:
+            cluster: default
+            user: gardenlet-bootstrap
+          name: gardenlet-bootstrap
+        users:
+        - name: gardenlet-bootstrap
+          user:
+            token: 07401d.f395accd246ae52d
+  seedConfig:
+    metadata:
+      # Set via skaffold.yaml
+      name: ""
+    spec:
+      backup: null
+      secretRef: null
+      dns:
+        provider:
+          secretRef:
+            # Set via skaffold.yaml
+            name: ""
+            namespace: garden
+          # Set via skaffold.yaml
+          type: ""
+      ingress:
+        controller:
+          kind: nginx
+        # Set via skaffold.yaml
+        domain: ""
+      networks:
+        blockCIDRs:
+        - 169.254.169.254/32
+        # Set via skaffold.yaml
+        pods: ""
+        # Set via skaffold.yaml
+        nodes: $NODES_CIDR
```

Comment:
Why do we need this? I don't see such variables anywhere else?

Reply:
This was a leftover.
I moved the entire `seedConfig` anyway to `example/provider-extensions/gardenlet/values.yaml.tmpl`.
Skaffold value templates do not work for lists, which are needed to set the `zones` of the seed. Thus, I removed the values templates and write the seedConfig to a values.yaml now. This is used in skaffold then.